fs/vfs/fs_eventfd.c

```diff
@@ -135,6 +135,7 @@ static FAR struct eventfd_priv_s *eventfd_allocdev(void)
 
       nxmutex_init(&dev->lock);
       nxmutex_lock(&dev->lock);
+      dev->crefs += 1;
```

Comment:
check the reference count do not overflow before alloc the resource

Reply:
No, because alloc dev is called when the user first calls eventfd, timer_create, signalfd, do not call it when dup.

Reply:
These API such as `eventfd_allocdev()` is a help function, from api design, you do not get to decide how developers use it. If it will alloc once on init phase, then you should move the increment of the reference count to `eventfd()` not `eventfd_allocdev()`